src/libs/actions/QueuedOnyxUpdates.js

```diff
@@ -14,7 +14,7 @@ Onyx.connect({
  * @returns {Promise}
  */
 function queueOnyxUpdates(updates) {
-    return Onyx.merge(ONYXKEYS.QUEUED_ONYX_UPDATES, updates);
+    return Onyx.set(ONYXKEYS.QUEUED_ONYX_UPDATES, [...queuedOnyxUpdates, ...updates]);
```

Comment:
Was this code just wrong before? I am confused about this change, what it does, and why it fixes anything.

Reply:
This is due to the changes in onyx regarding the merge behavior using JSON PATCH 

I think it was this PR https://github.com/Expensify/react-native-onyx/pull/284 @chrispader ?

Reply:
Oh got it. So yeah we were still using this as a way to concat items. Thanks, that's a great catch.